cf-custom-resources/lib/nlb-cert-validator-updater.js

```diff
@@ -0,0 +1,311 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+const AWS = require('aws-sdk');
+const ATTEMPTS_VALIDATION_OPTIONS_READY = 10;
+const ATTEMPTS_RECORD_SETS_CHANGE = 10;
+const DELAY_RECORD_SETS_CHANGE = 30;
+const ATTEMPTS_CERTIFICATE_VALIDATED = 19;
+const DELAY_CERTIFICATE_VALIDATED = 30;
+
+let acm, envRoute53, envHostedZoneID, appName, envName, certificateDomain;
```

Comment:
should we add serviceName here too ? since we pass it down to other functions

Reply:
As long as we don't allow changing the service name in the future!